init.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+DEPLOY_ENVIRONMENT="${DEPLOY_ENVIRONMENT:=dev}" 
+
```

Comment:
```suggestion
export ZKSYNC_HOME=$PWD
export PATH=$ZKSYNC_HOME/bin:$PATH
```
Please add the zk installation here